Author: Tom Jampen <tom@cryptography.ch>
Description:
 Patches texstudio.desktop to add important keywords. It also removes the
 useless categories.

diff -Naurp a/utilities/texstudio.desktop b/utilities/texstudio.desktop
--- a/utilities/texstudio.desktop
+++ b/utilities/texstudio.desktop
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Categories=Office;Publishing;Qt;X-SuSE-Core-Office;X-Mandriva-Office-Publishing;X-Misc;
+Categories=Office;Publishing;
 Exec=texstudio %F
 GenericName=LaTeX Editor
 GenericName[de]=LaTeX Editor
@@ -10,7 +10,7 @@ Comment[de]=LaTeX Entwicklungsumgebung
 Comment[fr]=Environnement de développement LaTeX
 Comment[ru]=Среда разработки LaTeX
 Icon=texstudio
-Keywords=LaTeX;TeX;editor;
+Keywords=editor;latex;pdflatex;xelatex;lualatex;context;bibtex;tex;
 MimeType=text/x-tex;
 Name=TeXstudio
 StartupNotify=false
